packages/wp-now/project.json

```diff
@@ -0,0 +1,74 @@
+{
+	"name": "wp-now",
+	"$schema": "../../node_modules/nx/schemas/project-schema.json",
+	"sourceRoot": "packages/wp-now/src",
+	"projectType": "application",
+	"targets": {
+		"build": {
+			"executor": "nx:noop",
+			"dependsOn": ["build:package-json", "build:bundle"]
+		},
+		"build:package-json": {
+			"executor": "@wp-playground/nx-extensions:package-json",
+			"options": {
+				"tsConfig": "packages/wp-now/tsconfig.lib.json",
+				"outputPath": "dist/packages/wp-now",
+				"buildTarget": "wp-now:build:bundle"
+			},
+			"dependsOn": ["build:bundle"]
+		},
+		"build:bundle": {
+			"executor": "@nrwl/workspace:run-commands",
+			"options": {
+				"commands": [
+					"node packages/wp-now/esbuild.mjs",
+					"cp packages/wp-now/package.json dist/packages/wp-now"
```

Comment:
why do we need to copy a `package.json` into `wp-now`?

Reply:
In the end, we are using default esbuild inside `nx` and all the packages need their `package.json` inside the dist folder.